org.solarus_games.solarus.Launcher.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.solarus_games.solarus.Launcher
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: solarus-launcher
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - glm.json
+  - org.luajit.Luajit.json
```

Comment:
Please merge these back to the main manifest.

Reply:
Can you elaborate why and how?

Reply:
1. open `org.luajit.Luajit.json` in text editor. Select all, copy. 
2. in the main manifest, Paste, Reindent as needed by YAML.

Reply:
But the `glm.json` is a shared module and LuaJIT probably should be one as well.

Reply:
It's not a shared module if it's not in the shared-modules repository.

Reply:
Looking at https://luajit.org/download.html I see there are conflicting versions and game makers attempting to use a development version. Looking at https://gitlab.com/solarus-games/solarus/-/blob/dev/compilation.md I was also packaging the wrong one. Probably not a good candiate for a shared version.